README.md

```diff
@@ -84,21 +84,16 @@ let dip = DependencyContainer<String> { dip in
 
 * If you give a `tag` in the parameter to `register()`, it will associate that instance or factory with this tag, which can be used later during `resolve` (see below).
 * `resolve(tag)` will try to find an instance (or instance factory) that match both the requested protocol _and_ the tag. If it doesn't find any, it will fallback to an instance (or instance factory) that only match the requested protocol.
-* The tags can be anything, as long as it's of a type conforming to `Equatable`. `DependencyContainer` is a generic class which takes that type as generic parameter (so one `DependencyContainer` will be tied with a given tag type)
+* The tags can be StringLiteralType or IntegerLiteralType. That said you can use plain strings or integers as tags.
 
-For example, you can use an `Int`, a `String`… or even an `enum`, like this:
 
 ```swift
-enum WebService {
-    case PersonWS
-    case StarshipWS
-}
+let PersonWSTag: Tag = "PersonWS"
```

Comment:
What about using `enum WebService : String`, and then use the `.rawValue` for the tag? This would make nice "scoped constants" instead of keeping them wild at the global namespace level with those top-level `let`?